### PR TITLE
Take into account the EOM of Swiftmailer in docs

### PR DESCRIPTION
**Pull request by fabpot**
See https://github.com/swiftmailer/swiftmailer/pull/1352 for the context

### CODE REVIEW

File: console/command_in_controller.rst [hunk 1]
```diff
@@ -20,11 +20,9 @@ Instead, you can run the command directly from the controller.
     a controller has a slight performance impact because of the request stack
     overhead.
 
-Imagine you want to send spooled Swift Mailer messages by
-:doc:`using the swiftmailer:spool:send command </email>`.
-Run this command from inside your controller via::
+Imagine you want to run the `debug:twig` from inside your controller::
```
Comment (fabpot) [inline console/command_in_controller.rst]: The example is not great, but frankly, I think we should remove this doc as you should never have to do this.

File: migration.rst [hunk 2]
```diff
@@ -82,7 +82,7 @@ Setting up Composer
 Another point you will have to look out for is conflicts between
 dependencies in both applications. This is especially important if your
 existing application already uses Symfony components or libraries commonly
-used in Symfony applications such as Doctrine ORM, Swiftmailer or Twig.
+used in Symfony applications such as Doctrine ORM  or Twig.
```
Comment (wouterj) [inline migration.rst]:
```suggestion
used in Symfony applications such as Doctrine ORM or Twig.
```

**Comment by fabpot** [comment]
When merging up, I propose to remove everything related to Swiftmailer in the 6.0 branch.

File: console/command_in_controller.rst (hunk 1 above)
Comment (OskarStark) [inline console/command_in_controller.rst]: Two backticks please

Comment (fabpot) [reply console/command_in_controller.rst]: Good catch, fixed

**Review by OskarStark — APPROVED** [review]

**Comment by fabpot** [comment]
More info here: More information here: https://symfony.com/blog/the-end-of-swiftmailer

Ready to be merged.

**Comment by OskarStark** [comment]
Thank you Fabien.

**Pull request merged**